flecs_ecs/src/core/entity_view/entity_view_const.rs

```diff
@@ -715,7 +717,9 @@ impl<'a> EntityView<'a> {
     ///
     /// # Returns
     ///
-    /// - If the callback has ran.
+    /// - If the callback returns `Option`, acts like [`and_then`](Option::and_then); returns the `Option` if the callback was run, `None` otherwise.
+    ///   If you need to differentiate between "None returned from callback" and "None because the callback was skipped", use the `Option` wrapper
```

Comment:
I'm having trouble understanding this, I request a doc test example :D

Reply:
Resolved by refactoring (and examples)